core/src/main/java/org/mule/runtime/core/policy/CompositeOperationPolicy.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.policy;
+
+import org.mule.runtime.api.exception.MuleException;
+import org.mule.runtime.core.api.DefaultMuleException;
+import org.mule.runtime.core.api.Event;
+import org.mule.runtime.core.api.policy.OperationPolicyParametersTransformer;
+import org.mule.runtime.core.api.policy.SourcePolicyParametersTransformer;
+import org.mule.runtime.core.api.processor.Processor;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * {@link OperationPolicy} created from a list of {@link ParameterizedPolicy}.
```

Comment:
It's an utility class that we have just to group a policy and it's parameters. It does not have any behaviour that we can benefit from having an interface, at least for now.